test/test_backbone_utils.py

```diff
@@ -28,16 +29,22 @@ def test_resnet_fpn_backbone(backbone_name):
         resnet_fpn_backbone(backbone_name, False, returned_layers=[0, 1, 2, 3])
     with pytest.raises(ValueError, match=r"Each returned layer should be in the range"):
         resnet_fpn_backbone(backbone_name, False, returned_layers=[2, 3, 4, 5])
+    model = resnet_fpn_backbone(backbone_name, False)
+    assert isinstance(model, BackboneWithFPN)
```

Comment:
I think this assertion is a good addition. To avoid reinstantiating the same variant, you can refactor line 23 above to return the `model` and have access to do the assertion prior the prediction.

Reply:
good catch. I will refactor.